eZ/Publish/Core/Repository/RoleService.php

```diff
@@ -183,6 +185,73 @@ public function createRoleDraft(APIRole $role): APIRoleDraft
         return $this->roleDomainMapper->buildDomainRoleDraftObject($spiRole);
     }
 
+    /**
+     * Copies an existing Role.
+     *
+     * @since 8.0
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to copy a Role
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if the name of the role already exists or if limitation of the same type
+     *         is repeated in the policy create struct or if limitation is not allowed on module/function
+     * @throws \eZ\Publish\API\Repository\Exceptions\LimitationValidationException if a policy limitation in the $roleCopyStruct is not valid
+     *
+     * @param \eZ\Publish\API\Repository\Values\User\Role $role
+     * @param \eZ\Publish\API\Repository\Values\User\RoleCopyStruct $roleCopyStruct
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\Role
+     */
```

Comment:
This is redundant, the same doc is on API interface, it should be there only (no need to inheritdoc, per the latest eZ Platform Coding Style).
```suggestion
```